pages/_app.tsx

```diff
@@ -2,10 +2,47 @@ import {ChakraProvider} from '@chakra-ui/react'
 import type {AppProps} from 'next/app'
 import {theme} from '../theme'
 
+import '@rainbow-me/rainbowkit/styles.css';
+import {
+  getDefaultWallets,
+  RainbowKitProvider,
+} from '@rainbow-me/rainbowkit';
+import {
+  chain,
+  configureChains,
+  createClient,
+  WagmiConfig,
+} from 'wagmi';
+import { alchemyProvider } from 'wagmi/providers/alchemy';
```

Comment:
Please configure for only the public provider and remove the alchemy provider configuration.